<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterfaceMidi</class>
 <widget class="QWidget" name="InterfaceMidi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="statusTip">
   <string>MIDI Interface|If you cannot use OpenSoundControl in your creative application, you can use the built in MIDI support of IanniX.\nOn Mac OS X, a new virtual MIDI device will be available; you just have to enable it in the settings of your software.\nIanniX is able to send notes-on/off, control changes, pitchbend... but it is not as simple to control IanniX with a MIDI surface control, we strongly recommend to use OpenSoundControl for that.</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QCheckBox" name="enable">
       <property name="toolTip">
        <string>Enables or disables MIDI messages</string>
       </property>
       <property name="text">
        <string>ENABLE MIDI</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="examples">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Opens the folder where you can find examples of IanniX interfacing with software using a MIDI connection</string>
       </property>
       <property name="text">
        <string>OPEN EXAMPLES</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_34">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="aliasLabel">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>DEFAULT MIDI OUT
(ALIAS MIDI_OUT)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>aliasPort</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="aliasPort">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Changes the default MIDI device for all outgoing MIDI messages</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_22">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>37</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_33">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="syncLabel">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>SEQUENCER
SYNCHRONIZATION</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>bpm</cstring>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <property name="spacing">
        <number>5</number>
       </property>
       <item>
        <widget class="QCheckBox" name="syncTransportIn">
         <property name="toolTip">
          <string>Check this option if you want to sync transport (play, pause, stop, rewind, song position) from another sequencer</string>
         </property>
         <property name="text">
          <string>SYNC TRANSPORT IN</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="syncTransportOut">
         <property name="toolTip">
          <string>Check this option if you want to sync transport (play, pause, stop, rewind, song position) to another sequencer</string>
         </property>
         <property name="text">
          <string>SYNC TRANSPORT OUT</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="bpm">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Changes the BPM of the score (used by clock synchronization)</string>
         </property>
         <property name="suffix">
          <string/>
         </property>
         <property name="prefix">
          <string>BPM @ </string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>65535</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_21">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="midiJack" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>You may need a virtual
MIDI jack to connect IanniX
with other MIDI software</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="download">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Downloads LoopBe1</string>
        </property>
        <property name="text">
         <string>DOWNLOAD LOOPBE1</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
